azure-pipelines.yml

```diff
@@ -0,0 +1,34 @@
+trigger:
+- main
+
+pool:
+  name: VSEngSS-MicroBuild2022-1ES
+
+steps:
+- task: UseDotNet@2
+  inputs:
+    packageType: 'sdk'
+    version: '7.0.100-preview.6.22352.1'
+    includePreviewVersions: true
+
+- task: MicroBuildSigningPlugin@4
+  inputs:
+    signType: 'test'
+    feedSource: 'https://devdiv.pkgs.visualstudio.com/DefaultCollection/_packaging/MicroBuildToolset/nuget/v3/index.json'
+
+- task: MSBuild@1
+  inputs:
+    solution: '**/*.sln'
+    msbuildArchitecture: 'x64'
+    configuration: 'Release'
+    msbuildArguments: '-restore -bl:$(Build.StagingDirectory)\ContainersOfficialBuild.binlog -v:minimal'
```

Comment:
does minimal verbosity affect the binlog?

Reply:
It does not